python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -359,7 +359,7 @@ def validate(self):
 
                 isJob = True
 
-                if (self.getJobObject().splitter != None):
+                if self.getJobObject().fqid.find('.') > -1:
```

Comment:
I think the way that a subjob is normally checked for in the Job class (and elsewhere) is to look at ```._master```.
Maybe a better test is:
```if self.getJobObject()._master is not None:```
Which should be equivalent.

I'll give either of these a test before the next 6.1.14 pre later tomorrow.